andela/strikers-ah-frontend
# fix bug to view unread notifications

#### What does this PR do?
This PR will help a user to differentiate between read notifications and unread notifications.
#### Description of Task to be completed?
- Added a section to check if a certain notification is read.
- Added tests for this feature.
#### How should this be manually tested?
- Clone the repo from https://github.com/andela/strikers-ah-frontend.git
- Got to where you cloned the repo and open a terminal/cmd there.
- Checkout to `develop` branch by typing `git checkout develop` in your terminal.
- Type `npm install` to install necessary dependencies.
- Then start the server with `npm start`.
- The home page will open up in your default browser and then you can register/login from the homepage.
- After, if there is any notification you will see them in the right corner of your page.
#### What are the relevant pivotal tracker stories?
#166826456
#### Screenshots (if appropriate).
<img width="1398" alt="Screen Shot 2019-06-21 at 04 19 58" src="https://user-images.githubusercontent.com/37753967/59893066-e1fbb080-93db-11e9-9cac-9a6b73f5e5e6.png">
